metabase/metabase
# Prototype model detail page

[Product doc](https://www.notion.so/metabase/Prototype-a-model-detail-page-94e07eac61d846b98e62bff4e1d82947)

Adds a basic and pretty raw prototype of a model detail page. The new page is available under the `/mode/:id/detail` route and should now become a page you see when clicking a model in a collection/search / recent items/bookmarks.

For now, it contains some general info (title, description, author, backing table) and two tabs:

* "used by" lists all the questions that use the model (brute-forced that on the FE for now)
* "schema" lists the fields used by a model

### To Verify

1. Create a model if you don't have one
2. Click it inside the collection / from search / from bookmark / from recent items
3. Ensure you're at `/mode/:id/detail` page
4. Ensure you can rename and change the description
5. Ensure the "Explore" button brings you to the chill mode
6. Ensure the default "Used by" tab lists questions using the model
7. Click "Schema" and ensure you see the model fields
8. Ensure "Edit metadata" brings you to the metadata editor

### Demo

<img width="1392" alt="CleanShot 2022-09-19 at 19 09 35@2x" src="https://user-images.githubusercontent.com/17258145/191084932-922c3050-b36a-492c-9dc2-7baf16e1a19d.png">
